src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -120,6 +120,11 @@
     "NAME": "Chill Touch",
     "DESCRIPTION": "Deal !D! damage to an enemy. NL It can no longer heal."
   },
+  "stsjorbsmod:Clarify": {
+    "NAME": "Clarify",
+    "DESCRIPTION": "Retain. While this is in your hand, Retain !M! extra card. NL stsjorbsmod:Exert. NL Gain Frozen Eye Effect.",
```

Comment:
There are enough cards with this pattern that we've started establishing a unique grammar for it diverged a bit from the base game to fit it better on the cards (see Frustration and Rebuttal). Let's do:

```Retain: While this is in your hand, Retain !M! extra card. NL stsjorbsmod:Exert: Your draw pile is shown in order.```